src/main/kotlin/commands/call_command.kt

```
@@ -34,7 +34,7 @@ class Call(
 ) : ListenerAdapter() {
 
     private val students = mutableSetOf<String>()
-    private val uniqueId = System.currentTimeMillis()
+    private val uniqueId = event.member!!.id + System.currentTimeMillis().toString()
```

No !!, cf. same issue in autoroles.kt ; use `let`

```kt
private val uniqueId = event.member?.let { it.id + System.currentTimeMillis().toString() } ?: ""
```

---

We can't use Elvis operator here, we absolutely need a unique ID based on requester.

---

It's better to throw an exception then (that is not an NPE ofc) ; you can throw it after the Elvis. Else you can do something like that maybe:

```kt
private val uniqueId = (event.member?.id ?: "00000000000000") + System.currentTimeMillis().toString()
```

---

Will change `member` to `user` to fix the issue and keep uniqueness of ID.